src/dash/student_view/TutorReviewModal.jsx

```diff
@@ -371,16 +371,18 @@ class TutorModal extends React.Component {
             fourth_star,
             fifth_star,
             satisfaction,
-        } = this.state;
-
-        const {
-            approval,
             request,
             topic,
             rejection_reason,
             rating,
             comment
+
+        } = this.state;
+
+        const {
+            approval,
         } = this.initTutorModal();
```

Comment:
since approval is equal to this function, it doesn't re-render when sockets changes the state of the modal when a request is approved

Reply:
Nice catch. for some reason, the function doesn't change the `approval` after searching... I'll try something else for Tuesday's meeting.

Reply:
sounds good!! sorry about all the bugs with this- I know my implementation was kind of messy